plugins/autowarmth.koplugin/main.lua

```diff
@@ -532,6 +533,8 @@ function AutoWarmth:getLocationMenu()
                 wrap = false,
                 value_step = 10,
                 value_hold_step = 100,
+                -- @translators This is the unit of length in meters.
+                unit = _("m"),
```

Comment:
Ah no, I meant more like this, sorry.
```suggestion
                unit = _C("Length", "m"),
```

Reply:
I have learn something new. Thanks

Reply:
Without a context all instances of a particular string are regarded as the same. With a context you can differentiate between different types of the same string. Some projects add a context to all strings.

In the translation interface the context can function similarly to a comment.